nnterp/tests/test_router.py

```
@@ -0,0 +1,346 @@
+import torch as th
+import pytest
+from nnterp import StandardizedTransformer
+from nnterp.rename_utils import RenamingError, RenameConfig
+from nnterp.tests.utils import TEST_MODELS
+
+
+def get_moe_models():
+    """Get list of MoE models for testing."""
+    moe_models = [
+        "yujiepan/mixtral-8xtiny-random",
+        "yujiepan/qwen1.5-moe-tiny-random", 
+        "yujiepan/qwen3-moe-tiny-random",
+    ]
+    # Filter to only include models that are in TEST_MODELS
+    return [model for model in moe_models if model in TEST_MODELS]
+
+
+def get_non_moe_models():
+    """Get list of non-MoE models for testing."""
+    non_moe_models = [
+        "gpt2",
+        "bigscience/bigscience-small-testing",
+        "yujiepan/opt-tiny-2layers-random",
+    ]
+    return [model for model in non_moe_models if model in TEST_MODELS]
```

same comment as above, instead of silently filtering we should throw an error